### ISSUE TITLE
Support of importing AAD groups directly to Azure DevOps group membership

### ISSUE THREAD

**Issue by rusticshiv86** [microsoft/terraform-provider-azuredevops#383]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Currently, we can easily create user entitlements and assign them to newly created local groups within Azure DevOps. However, it would be very effective if we could have something similar for groups too.

Use case - 

Create ADO Project => Create ADO group => import existing AAD group members ( or group itself) as "members" for created group.

Simply put => importing group from AAD directly into given project. Tried lot of ways already but did not help. 

### New or Affected Resource(s)

group and members resource as part of given provider

* azuredevops_group
* azuredevops_group_membership
* azuredevops_user_entitlement

### Potential Terraform Configuration

resource "azuredevops_group" "ado_gp" {
  scope = azuredevops_project.ado_prj.id
  display_name = "ado-gp-dev"
  description = "ADO Group for dev"

   members = [
     "aXXXXXXXXX"     <= existing group from aad not individual member/s
  ]
}

### References

* #0000

**Comment by xuzhang3** [comment]
@rusticshiv86 We have move this repo  to https://github.com/terraform-providers/terraform-provider-azuredevops.  Can you open another issue in the new repo?
Based on you description, I think what you need is a `azuredevops_membertship_data_resource` which can be used for importing exist AAD groups.  I need some investigation for this new resource.

Potential configure:
```
azuredevops_menbershiop {
     org_id/ project_id
     groups{ 
          {
            name: "admin"
            desc: "group description"
            members: {
                mem1,
                mem2,
            }
          }
    }
}
```

**Comment by rusticshiv86** [comment]
sure @xuzhang3 . Will move the issue there..

**Comment by rusticshiv86** [comment]
Hi @xuzhang3 , could you please try to transfer the issue to new repo yourself ? I guess I lack permissions to move it myself.

**Comment by xuzhang3** [comment]
@rusticshiv86  you can open a new issue in the new repo and close this one.

**Comment by rusticshiv86** [comment]
@xuzhang3  its done, new link https://github.com/terraform-providers/terraform-provider-azuredevops/issues/51 


-Shiva

**Comment by xuzhang3** [comment]
Closing this issue as this issue has been moved to the new repo.
